### DISCUSSION TITLE
Error during tranfer-learning process.

### DISCUSSION THREAD

**Discussion by Varuzhan97** [coqui-ai/STT#2038] [General]
I am trying transfer learning. The command is below.
`python -m coqui_stt_training.train --drop_source_layers 1 --epochs 20 --alphabet_config_path /home/varuzhan/Desktop/Arabic/Alphabet.txt --save_checkpoint_dir /home/varuzhan/Desktop/Arabic/checkpoint_ar --load_checkpoint_dir /home/varuzhan/Desktop/Arabic/checkpoint_en --train_files /home/varuzhan/Desktop/Arabic/Data/ar/clips/train.csv --dev_files /home/varuzhan/Desktop/Arabic/Data/ar/clips/dev.csv --test_files /home/varuzhan/Desktop/Arabic/Data/ar/clips/test.csv --export_dir /home/varuzhan/Desktop/Arabic/Model --train_batch_size 1 --dev_batch_size 1 --test_batch_size 1 --use_allow_growth true --train_cudnn true --max_to_keep 1`
And I am getting error like this:
`Traceback (most recent call last):
  File "/usr/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/usr/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/home/varuzhan/Desktop/Coqui/STT/training/coqui_stt_training/train.py", line 687, in <module>
    main()
  File "/home/varuzhan/Desktop/Coqui/STT/training/coqui_stt_training/train.py", line 643, in main
    initialize_globals_from_cli()
  File "/home/varuzhan/Desktop/Coqui/STT/training/coqui_stt_training/util/config.py", line 854, in initialize_globals_from_cli
    c = BaseSttConfig.init_from_argparse(arg_prefix="")
AttributeError: type object 'BaseSttConfig' has no attribute 'init_from_argparse'
`

**Comment by reuben** [comment]
Make sure you have all dependencies installed. You can re-run the installation step to make sure that's correct: https://stt.readthedocs.io/en/latest/TRAINING_INTRO.html#install-dependencies-and-stt

**Reply by mrinal-sourav** [reply]
It didn't work for me. It looks like the function "init_from_argparse" has not been defined; it occurs only once in the file @ line 854 here.
